### DISCUSSION TITLE
Failed to fetch https://ppa1.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu/dists/focal/main/binary-i386/Packages

### DISCUSSION THREAD

**Discussion by KaiyuWei** [rabbitmq/rabbitmq-server#8934] [Questions] [answered]
I ran `sudo apt update` and I found thatt this url is invalid, so my rabbitmq-server cannot be updated. Is there anything I can do to fix this?
```
E: Failed to fetch https://ppa1.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu/dists/focal/main/binary-i386/Packages  404  Not Found [IP: 172.67.140.63 443]
```

**Comment by michaelklishin** [comment, marked as answer]
The mirror does not have `i386` (32-bit) packages. We never produced them or published for 32-bit. Maybe Cloudsmith itself indexes arch-neutral packages for i386 but Erlang Debian packages are not arch-neutral (while RabbitMQ Debian packages are).

Use a 64-bit machine.

**Comment by michaelklishin** [comment]
Compare

```
curl https://ppa1.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu/dists/focal/main/binary-i386/Packages
```

to

```
curl https://ppa1.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu/dists/focal/main/binary-amd64/Packages
```

which does return a package index file.

**Reply by KaiyuWei** [reply]
Thanks! I think this is the problem for me. I'm indeed using a 64-bit devise, not sure how this url was generated. So how can I change the configuration so that `sudo apt get` fetches packages via the correct url? I checked `rabbitmq.list` file but I did not find the link the url for 32-bit so I don't know how to change it to the one for 64-bit

**Comment by Dawnthorn** [comment]
I was able to work around this issue by adding `arch=amd64` to all the deb lines lines in `/etc/apt/sources.list.d/rabbitmq.list`.  For example, this line:
```
deb [signed-by=/usr/share/keyrings/rabbitmq.E495BB49CC4BBE5B.gpg] https://ppa1.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu jammy main
```
Would change to this line:
```
deb [arch=amd64 signed-by=/usr/share/keyrings/rabbitmq.E495BB49CC4BBE5B.gpg] https://ppa1.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu jammy main
```

**Reply by michaelklishin** [reply]
Thank you for submitting https://github.com/rabbitmq/rabbitmq-website/pull/1971, it will be live in a few minutes.
